app/quests/helpers.py

```diff
@@ -46,6 +46,7 @@ def record_award_helper(qa, profile, layer=1, action='Beat', value_multiplier=1)
     """
     Awards point awards (and referral rewards) to a winner
     """
+    return  # temporarily disabled (until we solve the hijacking problem)
```

Comment:
The goal is to only disable the referral rewards, not all the rewards by commenting L90 & L91

Reply:
Fixed.

Reply:
Note I changed the network to `document.web3network`, because it is otherwise impossible to debug it on localhost. See also https://github.com/gitcoinco/web/pull/6585